lib/tapioca/dsl/compilers/controller_actions.rb

```diff
@@ -0,0 +1,75 @@
+# typed: strict
+# frozen_string_literal: true
+
+begin
+  require "action_controller"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Dsl
+    module Compilers
+      # `Tapioca::Dsl::Compilers::ControllerActions` generates RBI files for classes that include
+      # [`ActionController::Base`](https://api.rubyonrails.org/classes/ActionController/Base.html).
+      #
+      # For example, with the following class that includes `ActionController::Base`:
+      #
+      # ~~~rb
+      # class PostsController < ApplicationController
+      #  def index
+      #   # ...
+      #  end
+      #
+      #  def create
+      #    # ...
+      #  end
+      # end
+      # ~~~
+      #
+      # this compiler will produce the RBI file `posts_controller.rbi` with the following content:
+      #
+      # ~~~rbi
+      # # posts_controller.rbi
+      # # typed: strong
+      # class PostsController
+      #   sig { void }
+      #   def index; end
+      #
+      #   sig { void }
+      #   def create; end
+      # end
+      # ~~~
+      class ControllerActions < Compiler
+        extend T::Sig
+
+        ConstantType = type_member { { fixed: T.class_of(::ActionController::Base) } }
+
+        sig { override.void }
+        def decorate
+          public_methods = constant.public_instance_methods(false)
```

Comment:
Actually, I think we want to use `action_methods` instead, since it is what Rails uses internally: https://api.rubyonrails.org/v7.0.0/classes/AbstractController/Base.html#method-c-action_methods